KabuKit/Classes/core/Swift/Screen.swift

```diff
@@ -26,9 +26,9 @@ public protocol Screen : class {
      - Attention :
      ただし、前のシーンがない場合は戻らず、何も起きない
      */
-    func leave() -> Void
+    func leave(_ needsRewind: Bool) -> Void
```

Comment:
`needsRewind` ... 
umm.....
`leave` is close to meaning to `go back`, so `rewind` can't be separate with `leave`.
_(sorry,, Just my feeling, I can't explain well)_ 
I think this parameter called to be `needsTransition` or `useTransition` rather than `needsRewind`